scripts/index.js

```diff
@@ -52,6 +58,29 @@ module.exports = (robot) => {
     rooms.forEach(room => robot.messageRoom(room, message));
   };
 
+  // Notify HUBOT_IRC_ROOMS if someone files a blocker bug against BUGZILLA_BLOCKER_PRODUCT.
+  // Inform periodically until someone either takes the bug to fix it or downgrades its severity.
+  if (process.env.BUGZILLA_BLOCKER_PRODUCT) {
+    setInterval(async () => {
+      try {
+        const { body: { bugs } } = await got.get(`${BUGZILLA_BASE_URL}/rest/bug`, {
+          json: true,
+          query: {
+            product: process.env.BUGZILLA_BLOCKER_PRODUCT,
+            bug_severity: 'blocker',
+            resolution: '---',
+            bug_status: ['NEW', 'REOPENED'],
+            priority: ['--', 'P1', 'P2'],
+          },
+        });
+
+        bugs.map(bug => messageRooms(`[Blocker] ${bug.summary} https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}.`));
+      } catch (err) {
+        robot.logger.error(err);
+      }
+    }, process.env.BUGZILLA_BLOCKER_TIME_PERIOD || 5 * 60 * 1000);
```

Comment:
If you create a default `.env` file, you can set this default value there:

```
BUGZILLA_BLOCKER_TIME_PERIOD=300000
```

```js
parseInt(process.env.BUGZILLA_BLOCKER_TIME_PERIOD, 10)
```

Maybe rename to `BUGZILLA_BLOCKER_INTERVAL`.

Reply:
Very good idea. I will revise.